### PR TITLE
retro award removal rpip draft

### PR DESCRIPTION
**Pull request by shfryn**
RPIP draft to remove rerespective award cap outlined in RPIP-18

### CODE REVIEW

File: RPIPs/rpip-remove_retro_cap.md [hunk 1]
```diff
@@ -100,17 +98,75 @@ The motivation for the update in this RPIP is that:
...
+
+- After the interviews and evaluations are complete, the GMC will deliberate and make a decision regarding the candidate selection.
+- The decision will be based on the candidate's qualifications, experience, alignment with the organization's values, and their ability to fulfill the position's responsibilities effectively.
+- The decision will come within 14 days of the application window closing.
+
+##### Transition and Renewal of Administrator's Term:
+
+To ensure a smooth transition and maintain operational continuity, the committee will assess the administrator's term one month prior to its end. At that point, the committee will have two options:
+
+- Term Extension: The committee may choose to extend the current administrator's term by one year, provided they have been performing effectively and meeting the organization's expectations. This allows for continuity in their role and provides stability within the position.
```
[the remark this answers is not in the document]
Comment (shfryn) [reply RPIPs/rpip-remove_retro_cap.md:141]: Yes, the initial term is one year.

File: RPIPs/rpip-remove_retro_cap.md [hunk 2]
```diff
@@ -124,13 +131,14 @@ The administrator will follow an internal handbook that they manage for their pr
...
+- The Grants and Management Committee (GMC) will review the received applications once the application window closes.
 - The GMC will conduct interviews with shortlisted candidates to assess their suitability for the role.
   - The interviews may include a combination of virtual or in-person meetings, skill assessments, and reference checks.
 
 ##### Decision-Making
 
-- After the interviews and evaluations are complete, the GMC will deliberate and make a decision regarding the candidate selection.
+- After the interviews and evaluations are complete, the GMC will deliberate and make a decision regarding the candidate selection. The decision will be made through a single choice vote, where each member of the GMC selects their preferred candidate from the pool of applicants. The candidate who receives more than 50% of the votes cast by the GMC members will be selected. In the event that nobody meets the threshold (no candidate receives more than 50% of the votes), or if multiple candidates receive an equal number of votes, The GMC has 7 days to engage in further discussions to reevaluate the candidates, review their qualifications, and reconsider their merits.
```
Comment (Valdorff) [inline RPIPs/rpip-remove_retro_cap.md:140]: Better, but... what happens after those 7 days? Another majority vote? Is it any number of majority votes can happen? And what if nobody gets selected at the end?

File: RPIPs/rpip-remove_retro_cap.md [hunk 3]
```diff
@@ -150,20 +158,13 @@ To ensure a smooth transition and maintain operational continuity, the committee
 
 ###### Committee Member Replacement
 
-- Once the decision to remove the administrator has been made, a backup committee member will be assigned to assume the administrator's responsibilities temporarily during the hiring process.
+- If the GMC Administrator role is vacant at any time, a backup committee member will be assigned to assume the administrator's responsibilities temporarily during the hiring process. The backup will receive compensation equivalent to the administrator's compensation during the time the backup serves.
```
Comment (Valdorff) [inline RPIPs/rpip-remove_retro_cap.md:161]: I didn't notice this... but "will be assigned" isn't clear. Assigned by who and how? I might suggest making the GMC treasurer serve as the fallback simply to remove ambiguity or add a step that creates a time without anyone in the role.

File: RPIPs/rpip-remove_retro_cap.md [hunk 4]
```diff
@@ -158,7 +158,7 @@ To ensure a smooth transition and maintain operational continuity, the committee
 
 ###### Committee Member Replacement
 
-- If the GMC Administrator role is vacant at any time, a backup committee member will be assigned to assume the administrator's responsibilities temporarily during the hiring process. The backup will receive compensation equivalent to the administrator's compensation during the time the backup serves.
+- If the GMC Administrator role is vacant at any time, the GMC has seven days to discuss and assign a current GMC member to act as a backup and to assume the administrator's responsibilities temporarily during the hiring process. The backup will receive compensation equivalent to the administrator's compensation during the time the backup serves.
```
Comment (Valdorff) [inline RPIPs/rpip-remove_retro_cap.md:161]: Better, but not clear what the process is (majority? what happens if none chosen?). A suggestion:

```
If the GMC Administrator role is vacant at any time, the GMC SHALL, within at most 3 days, select a current GMC member to act as interim administrator during the hiring process. This selection is based on a vote, with the highest vote receiver being selected. In the event of a tie, the pDAO treasurer will cast the tie breaking vote. The backup will receive compensation equivalent to the administrator's compensation during the time the backup serves.
```

File: RPIPs/rpip-remove_retro_cap.md (hunk 3 above)
Comment (Valdorff) [reply RPIPs/rpip-remove_retro_cap.md]: resolving as newer comment exists

**Review by Valdorff — APPROVED** [review]
Draft level ready 👍

**Pull request merged**